app/models/manageiq/providers/azure/inventory/collector/target_collection.rb

```diff
@@ -52,7 +52,7 @@ def flavors
     set = Set.new(refs)
 
     collect_inventory_targeted(:series) { @vmm.series(@ems.provider_region) }.select do |flavor|
-      set.include?(flavor.name.downcase)
```

Comment:
I think this should stay and we should use ```refs = references(:flavors)``` on L49

Reply:
Ok, I've made that change.